### ISSUE TITLE
 ExtractorError 'Cannot decrypt signature without player_url'

### ISSUE THREAD

**Issue by droidzone** [rg3/youtube-dl#5485]
Version: 2015.04.17
OS: Debian 7 64bit

Code:

```
# youtube-dl -f 141 https://www.youtube.com/watch?v=6PGKODQcOw8 --verbose
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-f', u'141', u'https://www.youtube.com/watch?v=6PGKODQcOw8', u'--verbose']
[debug] Encodings: locale ANSI_X3.4-1968, fs ANSI_X3.4-1968, out ANSI_X3.4-1968, pref ANSI_X3.4-1968
[debug] youtube-dl version 2015.04.17
[debug] Python version 2.7.3 - Linux-2.6.32-042stab102.9-x86_64-with-debian-7.8
[debug] exe versions: none
[debug] Proxy map: {}
[youtube] 6PGKODQcOw8: Downloading webpage
[youtube] 6PGKODQcOw8: Extracting video information
WARNING: [youtube] 6PGKODQcOw8: Skipping DASH manifest: ExtractorError(u'Cannot decrypt signature without player_url; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.',)
ERROR: requested format not available
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 662, in extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 708, in process_ie_result
    return self.process_video_result(ie_result, download=download)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1147, in process_video_result
    expected=True)
ExtractorError: requested format not available
```

**Comment by yan12125** [comment]
Could you run the follow command:
```
youtube-dl --write-pages "https://www.youtube.com/watch?v=6PGKODQcOw8" --verbose
```
And paste the content of the dump file ```6PGKODQcOw8_https_-_www.youtube.com_watchv=6PGKODQcOw8_gl=US_hl=en_has_verified=1_bpctr=9999999999.dump``` to gist or pastebin or other pasting services?

The format 141 comes from the DASH manifest, but youtube-dl can't handle it due to missing decryption function. Try other format codes from the listing given by ```-F``` option.

**Comment by yan12125** [comment]
This is most likely fixed by a49eccd. Feel free to open a new issue if it's still broken in the next version.
